api/agentpb/agent.proto

```diff
@@ -167,6 +167,14 @@ message StartActionRequest {
   // PTSummaryParams describes parameters for PT summary.
   message PTSummaryParams {}
 
+  // PTMySQLSummaryParams describes parameters for PT MySQL summary.
+  message PTMySQLSummaryParams {
+    string addr = 1;
```

Comment:
why are did you cut field name?

Reply:
I just did. Was there any requirement specifying the members' names?